common/src/main/AndroidManifest.xml

```diff
@@ -1,22 +1,26 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.microsoft.identity.common">
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
+    <uses-sdk tools:overrideLibrary="com.yubico.yubikit.android" />
+
     <application>
+        <!-- Because Android detects plugged in YubiKeys as keyboard config changes, configChanges must have `keyboardHidden` and `keyboard` -->
         <activity
             android:name="com.microsoft.identity.common.internal.providers.oauth2.AuthorizationActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout" />
+            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout|keyboard" />
```

Comment:
Can you share official Android documentation on what this does? I think you said in the description that Android handles this themselves but not entirely clear what Android does here

Reply:
https://developer.android.com/guide/topics/resources/runtime-changes#HandlingTheChange
The default handling of config changes is that the activity is destroyed and created again. I didn't want this to happen whenever a YubiKey was plugged in, hence the addition of the `keyboard` flag, so that nothing happens instead (until we're able to start up the YubikitManger in the EoClientCertAuthChallengeHandler).